### ISSUE TITLE
Add 2FA support

### ISSUE THREAD

**Issue by SuperSandro2000** [majewsky/portunus#26]
It would be nice if we would have 2FA for more security. This wouldn't work over LDAP, so just the Web UI could be protected which would already be a win for admins. Additionally enforcing this for portunus admins would be nice.

**Comment by majewsky** [comment]
I agree. I will probably not have the time to work on this in the short or midterm, but if anyone wants to work on it, you have my support. Please keep in touch and prefer to submit PRs early so we can align on the design and structure.

**Comment by SuperSandro2000** [comment]
Please take a look at #27.
